experiments/rew_configs/Humanoid.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Well the way the weights are used, this would be the same reward scale as native to the Humanoid environment. This is the scale I've been using for all experiments except the very beginning where we experimented with having dense reward be 10 times as much.

Reply:
OK, but `run_humanoid.py` in Baselines repo downscales all the reward by 10x right? So we may want to do the same? I'd look empirically at the reward curves to see what scale they are and decide if we need to do this.